CONTRIBUTING.md

```diff
@@ -48,7 +52,7 @@ Netdata is all about simplicity and meaningful presentation. A "sponsor" for a c
 
 #### Sponsor a backend
```

Comment:
We should update this to mention and point to the exporting engine, not backends.

```suggestion
#### Sponsor an exporting connector
```